lib/safe_navigation_cop.rb

```diff
@@ -0,0 +1,18 @@
+module RuboCop::Cop
```

Comment:
Any reason to not use rubocop-rails' [SafeNavigation cop](https://github.com/rubocop-hq/rubocop-rails/blob/4712f515c67efc993ca8c6f62943bb761ff0b5df/lib/rubocop/cop/rails/safe_navigation.rb)?

Reply:
We don't have `rubocop-rails` in our Gemfile. We can add it someday. One would have to go through default checks and fix offences.